### DISCUSSION TITLE
Issue with submitting student application

### DISCUSSION THREAD

**Discussion by RickDoesStuff** [community/community#69714] [GitHub Education]
### Hi everyone,

Hi, I am trying to submit my application for GitHub student but every time I click process application, it loads and then doesn't do anything when it's done. It keeps saying I already submitted 2 requests and won't add my new one to that list. My application was rejected 8 days ago but I have fixed all those issues with my billing and email. I tried to submit it 7 days ago and a couple of times today and nothing has changed. I hope to get this figured out quickly since I use PyCharm regularly for class and it will expire soon so I need it to be renewed with my new school.

After the file was done uploading, iI pressed process and it became 100%, and the site refreshed and went back to its initial state.

I typically use Chrome, I tried Edge, I downloaded firefox and opera gx, and none of those worked.

**Comment by RhymeTheRapper** [comment]
I am having the same issue
